styles/styles.css

```diff
@@ -10,44 +10,40 @@
  * governing permissions and limitations under the License.
  */
 
- @font-face {
-  font-family: AvenirNextLTPro-Demi;
-  src: url("/fonts/AvenirNextLTPro-Demi.woff2") format("woff2");
-  font-display: swap;
-  font-style: normal;
-  font-weight: 400;
+
+/* fallback font for AvenirNextLTPro-Demi (normal - 400) */
+@font-face {
+  font-family: "avenirnextltpro-demi-normal-400-fallback";
+  size-adjust: 103.741%;
+  src: local("Helvetica");
```

Comment:
Not sure if we should use Helvetica as a fallback font. Helvetica is not a default font on Windows.